en/flight_controller/snapdragon_flight_hardware_example_setup.md

```diff
@@ -0,0 +1,57 @@
+# Snapdragon Hardware Setup Example
+
+This guide documents in detail one particular build based on the Snapdragon Flight platform. The PX4 development team has built, tested and documented this specific setup.
+
+![](../../assets/hardware/snapdragon/snapdragon_minimal_finished.jpg)
+
+
+## PX4 Dev Team Recommended Setup
```

Comment:
I would remove this heading and move all the others up one level. You've already stated that this is the recommended setup so no need to duplicate.

If it looks odd having the text below the diagram then I would change this text to "Overview" and again make all the other headings level 2 as well.

Reply:
done